### PR TITLE
Add a French version of Secret concept page

### PR DESCRIPTION
**Pull request by remyleone**

### CODE REVIEW

File: content/fr/docs/concepts/configuration/secret.md [hunk 1]
```diff
@@ -0,0 +1,980 @@
...
+  title: Gestion du secret et de la configuration
+  description: >
+    Déployez et mettez à jour les secrets et la configuration des applications sans reconstruire votre image et sans dévoiler les secrets de la configuration de vos applications.
+weight: 50
+---
+
+
+{{% capture overview %}}
+
+Les objets `secrets` de Kubernetes vous permettent de stocker et de gérer des informations sensibles, telles que les mots de passe, les jetons OAuth et les clés ssh.
```
Comment (sftim) [inline content/fr/docs/concepts/configuration/secret.md]: nit:
```suggestion
Les objets `secret` de Kubernetes vous permettent de stocker et de gérer des informations sensibles, telles que les mots de passe, les jetons OAuth et les clés ssh.
```
The actual name for the object is Secret but the upstream localization (English) puts it in the wrong case&mdash;and in backticks, which is different from the recommended way to write API object names.

File: content/fr/docs/concepts/configuration/secret.md [hunk 2]
```diff
@@ -0,0 +1,980 @@
...
+  * Si vous exécutez etcd dans un cluster, les administrateurs doivent s'assurer d'utiliser SSL/TLS pour la communication peer-to-peer etcd.
+* Si vous configurez le secret via un fichier manifeste (JSON ou YAML) qui a les données secrètes codées en base64, partager ce fichier ou l'archiver dans un dépot de source signifie que le secret est compromis.
+  L'encodage Base64 _n'est pas_ une méthode de chiffrement et est considéré comme identique au texte brut.
+* Les applications doivent toujours protéger la valeur du secret après l'avoir lu dans le volume, comme ne pas le mettre accidentellement dans un journal ou le transmettre à une partie non fiable.
+* Un utilisateur qui peut créer un pod qui utilise un secret peut également voir la valeur de ce secret.
+  Même si la stratégie apiserver ne permet pas à cet utilisateur de lire l'objet secret, l'utilisateur peut exécuter un pod qui expose le secret.
+* Actuellement, toute personne disposant des droit root sur n'importe quel nœud peut lire _n'importe quel_ secret depuis l'apiserver, en usurpant l'identité du kubelet.
```
[the remark this answers is not in the document]
Comment (remyleone) [reply content/fr/docs/concepts/configuration/secret.md]: Oui du verbe usurper

File: content/fr/docs/concepts/configuration/secret.md [hunk 3]
```diff
@@ -0,0 +1,980 @@
...
+```
+
+{{< note >}}
+Le nom des secrets généré a un suffixe ajouté en hachant le contenu.
+Cela garantit qu'un nouveau secret est généré chaque fois que le contenu est modifié.
+{{< /note >}}
+
+#### Décoder un secret
+
+Secrets can be retrieved via the `kubectl get secret` command. For example, to retrieve the secret created in the previous section:
```
Comment (AWKIF) [inline content/fr/docs/concepts/configuration/secret.md]: A traduire ?

File: content/fr/docs/concepts/configuration/secret.md [hunk 4]
```diff
@@ -0,0 +1,980 @@
...
+Un conteneur utilisant un secret comme un volume [subPath](/docs/concepts/storage/volumes#using-subpath) monté ne recevra pas de mises à jour secrètes.
+{{< /note >}}
+
+### Utilisation de secrets comme variables d'environnement
+
+Pour utiliser un secret dans une {{< glossary_tooltip text="variable d'environnement" term_id="container-env-variables" >}} dans un pod:
+
+1. Créez un secret ou utilisez-en un déjà existant.
+   Plusieurs pods peuvent référencer le même secret.
+1. Modifiez la définition de votre pod dans chaque conteneur où vous souhaitez utiliser la valeur d'une clé secrète pour ajouter une variable d'environnement pour chaque clé secrète que vous souhaitez consommer.
```
Comment (AWKIF) [inline content/fr/docs/concepts/configuration/secret.md]: 2.

File: content/fr/docs/concepts/configuration/secret.md [hunk 5]
```diff
@@ -0,0 +1,980 @@
...
+   La variable d'environnement qui consomme la clé secrète doit remplir le nom et la clé du secret dans `env[].valueFrom.secretKeyRef`.
+1. Modifiez votre image et/ou votre ligne de commande pour que le programme recherche des valeurs dans les variables d'environnement spécifiées
```
Comment (AWKIF) [inline content/fr/docs/concepts/configuration/secret.md]: 3.

File: content/fr/docs/concepts/configuration/secret.md [hunk 6]
```diff
@@ -0,0 +1,980 @@
...
+Ils peuvent également être utilisés par d'autres parties du système, sans être directement exposés aux Pods.
+Par exemple, ils peuvent détenir des informations d'identification que d'autres parties du système doivent utiliser pour interagir avec des systèmes externes en votre nom.
+
+### Utilisation de secrets comme fichiers d'un pod
+
+Pour consommer un secret dans un volume dans un pod:
+
+1. Créez un secret ou utilisez-en un déjà existant.
+   Plusieurs Pods peuvent référencer le même secret.
+1. Modifiez la définition de votre Pod pour ajouter un volume sous `.spec.volumes[]`.
```
Comment (AWKIF) [inline content/fr/docs/concepts/configuration/secret.md]: 2.

File: content/fr/docs/concepts/configuration/secret.md [hunk 7]
```diff
@@ -0,0 +1,980 @@
...
+   Nommez le volume et ayez un champ `.spec.volumes[].secret.secretName` égal au nom de l'objet secret.
+1. Ajouter un `.spec.containers[].volumeMounts[]` à chaque conteneur qui a besoin du secret.
```
Comment (AWKIF) [inline content/fr/docs/concepts/configuration/secret.md]: 3.

File: content/fr/docs/concepts/configuration/secret.md [hunk 8]
```diff
@@ -0,0 +1,980 @@
...
+   Spécifier `.spec.containers[].volumeMounts[].readOnly = true` et `.spec.containers[].volumeMounts[].mountPath` à un nom de répertoire inutilisé où vous souhaitez que les secrets apparaissent.
+1. Modifiez votre image et/ou votre ligne de commande pour que le programme recherche les fichiers dans ce répertoire.
```
Comment (AWKIF) [inline content/fr/docs/concepts/configuration/secret.md]: 4.

File: content/fr/docs/concepts/configuration/secret.md [hunk 9]
```diff
@@ -0,0 +1,980 @@
...
+  volumes:
+  - name: foo
+    secret:
+      secretName: mysecret
+      items:
+      - key: username
+        path: my-group/my-username
+```
+
+What will happen:
```
Comment (perriea) [inline content/fr/docs/concepts/configuration/secret.md]:
```suggestion
Que se passera-t-il:
```

**Pull request merged**